src/components/templates/PosterPage/components/PosterPageSettingsControl/PosterPageSettingsControl.scss

```diff
@@ -0,0 +1,15 @@
+.PosterPageSettingsControl {
+  &__popover-content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 100%;
+    width: 100%;
+  }
+}
+
+#poster-page-settings-popover {
+  // This must overwrite default popover height
+  height: 100px;
+}
```

Comment:
There are two ways of overwriting styles for this one. Either `!important`, or `id`. I didn't see the exact solution for this in your previous comments, other than `!important` is not a good solution. I couldn't find a way to increase the specificity using the classes

Reply:
> There are two ways of overwriting styles for this one. Either `!important`, or `id`

I don't believe that is true.

> I didn't see the exact solution for this in your previous comments

It was linked from the page I linked you. I detailed it more explicitly in one of my comments within this review I believe; including linking to this reference card (I don't have the other link on hand):

- https://specifishity.com/

> I couldn't find a way to increase the specificity using the classes

Is that after using the reference sheet shown above?

Reply:
A bit better approach https://github.com/sparkletown/sparkle/pull/1299/commits/7b6bff6a88a62989785e33270a386f7502f86846

Reply:
Seems like a decent enough solution for now.

Probably worth adding a comment to it to explain why it differs from our default patterns (because it's needed to increase the css specificity to override the other styles) so that someone doesn't inadvertently delete/change it in future.

Reply:
Added in https://github.com/sparkletown/sparkle/pull/1299/commits/9acbbec8f58d7d7b122f095b27d50c10400627c7